src/common/Constants.ts

```diff
@@ -317,6 +326,7 @@ export const SUPPORTED_TOKENS: { [chainId: number]: string[] } = {
   10: ["DAI", "SNX", "BAL", "WETH", "USDC", "POOL", "USDT", "WBTC", "UMA", "ACX"],
   137: ["USDC", "USDT", "WETH", "DAI", "WBTC", "UMA", "BAL", "ACX", "POOL"],
   324: ["USDC", "USDT", "WETH", "WBTC", "DAI"],
+  1135: ["WETH", "LSK", "USDC", "USDT"],
```

Comment:
AFAIK USDC isn't available until August, and LSK is still pending in a PR in constants, so we can't use it yet.

(In general we might consider deferring this line, since we also can't resolve token addresses on Lisk yet.)

```suggestion
  1135: ["WETH", "USDT"],
```

Reply:
Are we not going to support USDC.e? I removed them here: https://github.com/across-protocol/relayer/pull/1639/commits/2f3e078bc64e9c8596cfc2337731575bd0157a02